rancid-tomatillos/src/App/App.js

```diff
@@ -0,0 +1,75 @@
+import React, {Component} from "react";
+import {
+    BrowserRouter as Router,
+    Switch,
+    Route,
+    Redirect,
+} from "react-router-dom";
+import Dashboard from "../Dashboard/Dashboard";
+import Header from "../Header/Header";
+import Login from "../Login/Login";
+import ShowPage from "../MoviePage/MoviePage";
+
+import "./App.css";
+
+class App extends Component {
+    constructor() {
+        super();
+        this.state = {
+            loggedIn: false,
+            movies: null,
```

Comment:
We should change `this.state.movies` initial state to empty array instead of null -- think that'd help in case of errors, it won't give isssues due to`null` when it's looking for pieces of info from each movie to populate cards?